chrome/fed-modules.json

```diff
@@ -463,13 +463,13 @@
             }
         ]
     },
-    "rhosakUi": {
-        "manifestLocation": "/apps/rhosak-ui/fed-mods.json",
```

Comment:
@Hyperkid123 @karelhala is this the internal CDN path where the assets are copied from the github build-repo to akamai/fakamai? How do we know where does our assets lie in akamai/fakamai storage? which configuration decides this location?